PyInstaller/loader/rthooks.dat

```diff
@@ -24,4 +24,5 @@
     'usb':        ['pyi_rth_usb.py'],
     'win32com':   ['pyi_rth_win32comgenpy.py'],
     'cryptography': ['pyi_rth_cryptography.py'],
+    'multiprocessing': ['pyi_rth_multiprocessing.py'],
```

Comment:
What do you think about moving this data-struct into `PyInstaller.loader__init__.py` and add this entry only on darwin?